plugin/trino-bigquery/README.md

```diff
@@ -0,0 +1,29 @@
+# BigQuery Connector Developer Notes
+
+The BigQuery connector module has both some unit tests and some integration tests.
+The integration tests require access to a BigQuery instance in Google Cloud seeded with TPCH data.
+You can follow the steps below to be able to run the integration tests locally.
+
+## Requirements
+
+* Access to a Google Cloud account.
```

Comment:
Can you shed a light how one can get that account? Also what that account need to be (any requirements)?